developer/web-services/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Unleash the power of the SGID with the AGRC Web API
+title: Unleash the power of the SGID with the AGRC Web APIs
```

Comment:
I think api should be singular since that has become the product name. It isn't a description of the services being offered.

Reply:
Yeah, this is something I've wondered about. I think I was mostly trying to be consistent, and I thought it sounded off to say "The main API that we offer is geocoding and searching." in the "Possibilities" section. I'm not quite sure how to fix this issue....Would it make sense to word it more like, "The main services of our web API are geocoding and searching." throughout the article?

Reply:
oh, now I see, yes, I believe your statement is true about the main services of the web api